### PR TITLE
Add newsletter sec

### PR DESCRIPTION
**Pull request by srikant96**
Screenshot (89)

### CODE REVIEW

**Review by emphay — COMMENTED** [review]
LGTM - Looks Good to Me

**Comment by emphay** [comment]
Add Screenshot of mobile Version

File: blogs/01/styles/newsletter-section.css [hunk 1]
```diff
@@ -1,79 +1,40 @@
...
+	display:flex;
+margin:0px auto;
 }
-input
-{
-	border:#1b1bc3;
+ div.input-box > input.submit {
+	height:43px;
+	width:97px;
+	background-color:#2f89fc;
```
Comment (itsashishbharti) [inline blogs/01/styles/newsletter-section.css:31]: use variable name

File: blogs/01/styles/newsletter-section.css [hunk 2]
```diff
@@ -1,79 +1,40 @@
...
+    font-size: 30px;
     color: black;
     font-weight: 400;
     text-align: center;
 }
-.newsletter-section > div > p
-{
+div.box>p{
     font-size: 16px;
     color: #808088;
```
Comment (itsashishbharti) [inline blogs/01/styles/newsletter-section.css:19]: use variable name

File: blogs/01/styles/newsletter-section.css [hunk 3]
```diff
@@ -1,79 +1,40 @@
...
     width: 475px;
     height: 185px;
     margin: 75px auto;
 }
-.newsletter-section > div > h4
-{
-    font-size: 20px;
+div.box>h2{
+    font-size: 30px;
     color: black;
```
Comment (itsashishbharti) [inline blogs/01/styles/newsletter-section.css:13]: use variable name

File: blogs/01/styles/newsletter-section.css [hunk 4]
```diff
@@ -1,79 +1,40 @@
...
-    border: none;
-    transition: all,0.3s,ease; 
-    text-align: center;
- }
-.subscribe-section > form > button:hover
-{
-    background-color:#1c6dd7 ;
-    color: white;
- }
- @media screen and (max-width:425px)
```
Comment (itsashishbharti) [inline blogs/01/styles/newsletter-section.css:61]: please  write mobile css

**Comment by amezng** [comment]
not as per the design.

**Pull request closed without merging**